docs/UsersGuide/source/BuildingRunningTesting/VXCases.rst

```diff
@@ -93,7 +93,7 @@ First, navigate to the ``ufs-srweather-app/ush`` directory. Then, load the workf
 
 Users running a csh/tcsh shell would run ``source /path/to/etc/lmod-setup.csh <platform>`` in place of the first command above. 
 
-After loading the workflow, users should follow the instructions printed to the console. Usually, the instructions will tell the user to run ``conda activate regional_workflow``. 
+After loading the workflow, users should follow the instructions printed to the console. Usually, the instructions will tell the user to run ``conda activate srw_app``.
```

Comment:
@christinaholtNOAA This is another one we should probably leave as-is. The VX chapter always uses the most recent release code because the develop branch is always changing, whereas the release provides a consistent baseline. Since the v2.2.0 release will have workflow_tools, this chapter probably should, too. (The SI team is about to do an update to bring it in line w/v2.2.0, so it won't say regional_workflow much longer).